src/screens/bolt/Bolt.module.scss

```diff
@@ -98,13 +102,26 @@
     flex-grow: 1;
     text-align: left;
 
-    img { margin-right: 1rem; }
+    img {
+      margin-right: 1rem;
+    }
   }
 
   .next {
     flex-grow: 1;
     text-align: right;
 
-    img { margin-left: 1rem; }
+    img {
+      margin-left: 1rem;
+    }
   }
-}
\ No newline at end of file
+}
+
+.refreshIcon {
+  margin-left: 1rem;
+  cursor: pointer;
+}
+
+.refreshing {
+  animation: rotate 1s linear infinite;
```

Comment:
@usama-mk, you should test your changes at least. Rotate animation wasn't defined or imported.